src/WebJobs.Script.WebHost/Diagnostics/SystemTraceWriter.cs

```diff
@@ -53,6 +54,11 @@ public override void Trace(TraceEvent traceEvent)
                     functionName = value.ToString();
                 }
 
+                if (traceEvent.Properties.TryGetValue(ScriptConstants.TracePropertyHostVersionKey, out value) && value != null)
```

Comment:
I'd just keep it simple and use ScriptHost.Version where needed, w/o adding to the trace event